src/py-opentimelineio/opentimelineio/core/__init__.py

```diff
@@ -54,28 +59,77 @@
     'flatten_stack',
     'install_external_keepalive_monitor',
     'instance_from_schema',
-    'register_serializable_object_type',
-    'register_upgrade_function',
     'set_type_record',
     'add_method',
     'upgrade_function_for',
+    'downgrade_function_from',
     'serializable_field',
     'deprecated_field',
     'serialize_json_to_string',
     'serialize_json_to_file',
-    'register_type'
+    'register_type',
+    'type_version_map',
+    'release_to_schema_version_map',
 ]
 
 
-def serialize_json_to_string(root, indent=4):
-    return _serialize_json_to_string(_value_to_any(root), indent)
+def serialize_json_to_string(root, schema_version_targets=None, indent=4):
+    """Serialize root to a json string.  Optionally downgrade resulting schemas
+    to schema_version_targets.
 
+    :param SerializableObject root: root object to serialize
```

Comment:
The type needs to be the FQDN, i.e. `opentimelineio.core.SerializableObject`. Also you might have to put the type on a separate `:type root: ...` line (sometimes Sphinx can get unhappy with "complex" types when inlned with the `:param:` instruction.

Reply:
This type was linking up - I suspect because SerializableObject is in this module?  In any case, I can fully resolve it anyway, if it helps readability

Reply:
Indeed! https://opentimelineio--1387.org.readthedocs.build/en/1387/api/python/opentimelineio.core.html#opentimelineio.core.serialize_json_to_file

Ignore this comment then.